src/Container/Container.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
The improved `Container` class need to have the unit tests.

We have to make sure that this `Container` class should be worked as our expected behavior.

Reply:
yes